envoy/listener.go

```diff
@@ -0,0 +1,45 @@
+package envoy
+
+import (
+	"github.com/Nitro/sidecar/catalog"
+)
+
+const (
+	listenerEventBufferSize = 100
```

Comment:
100 is 5 times larger than what we have for the URL Listener. Happy to adjust this, depending on what your thoughts are in the [other thread](https://github.com/Nitro/sidecar/pull/52#discussion_r370935055).